docs/installing-solidity.rst

```diff
@@ -83,14 +83,20 @@ and runs it in a new container, passing the ``--help`` argument.
 
     docker run ethereum/solc:stable --help
 
-For example, You can specify release build versions in the tag for the 0.5.4 release.
+You can specify release build versions in the tag. For example:
 
 .. code-block:: bash
 
-    docker run ethereum/solc:0.5.4 --help
+    docker run ethereum/solc:stable --help
+
+Note
+
+Specific version tags are supported for the Docker tag such as `docker run ethereum/solc:0.8.23 --help`. We will be passing the
```

Comment:
Maybe this is sufficient? I don't think we need to repeat the whole command.

```suggestion
Specific compiler versions are supported as the Docker image tag such as `ethereum/solc:0.8.23`. We will be passing the
```

Reply:
Thanks @r0qs! SGTM :)